app/base/static/assets/js/vis/simple-bar-chart.js

```diff
@@ -1,7 +1,12 @@
 class SimpleBarChart {
+    /*
     CHART_WIDTH = 1000;
     CHART_HEIGHT = 600;
-    GAP = 10;
+    */
+
+    CHART_WIDTH = document.getElementById('charts').offsetWidth - 50;
+    CHART_HEIGHT = window.innerHeight - 120;
+    GAP = 20;
```

Comment:
can you comment on those magic numbers 50, 120, 20?

Reply:
get rid of these magic numbers, and recalculated a value and created a static number at common.js